.github/workflows/ci.yml

```
@@ -52,6 +52,227 @@ jobs:
+            echo "skip_review=true" >> $GITHUB_OUTPUT
+          else
+            echo "skip_review=false" >> $GITHUB_OUTPUT
+            echo "Provider check passed:"
+            echo "  - Bedrock: ${{ env.BEDROCK_CONFIGURED }}"
+            echo "  - Anthropic: ${{ env.ANTHROPIC_KEY_SET }}"
+          fi
+
+      - name: Get PR context
+        if: steps.validate_providers.outputs.skip_review != 'true'
+        id: pr_context
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)